docker-bake.hcl

```diff
@@ -459,6 +467,23 @@ target "uid-allocator" {
 }
 
 
+# Plugin Runtime
+# ----------------------------------------------------------------------
+
+# This is the Docker image that will host our Generators and Analyzers
+# until we swap over to Firecracker.
+target "docker-plugin-runtime" {
+  inherits   = ["_grapl-base"]
+  context    = "."
```

Comment:
Not a huge deal at this point, but it may be a good idea to create a separate directory for this, just for general context management / hygiene. So rather than `Dockerfile.plugin-runtime`, we'd have `plugin-runtime/Dockerfile`, with `plugin-runtime` as the context.

Again, no big deal now, particularly given the image doesn't really do anything yet 😅

Reply:
I'm amenable to that.